src/queries/options.ts

```diff
@@ -0,0 +1,8 @@
+import { NormalizerFn } from '../matches';
+
+export type BaseOptions = { hidden?: boolean };
```

Comment:
naming: Sorry for back and forth with the naming, I think we should add `query` part to the type name, as `BaseOptions` used through the code seems too generic, and also `Common` seems more appropriate, as `Base` suggest a base type for all options, but in many cases this is also the actual type, so `Common` seems more appropriate.

```suggestion
export type CommonQueryOptions = { hidden?: boolean };
```

Reply:
indeed makes sense